src/components/Buttons.js

```diff
@@ -4,20 +4,200 @@
  * Created: 3/29/23
  * Author: Abdullah Al Mamun <mamun1214@gmail.com>
  ****************************************** */
-import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+import InputFied from './Input';
 
-function Buttons({ label }) {
+function Buttons() {
+  const [calObj, setCalObj] = useState({
+    total: 0,
+    next: 0,
+    operation: null,
+  });
   return (
-    <button
-      type="button"
-      className="cal-btn"
-    >
-      {label}
-    </button>
+    <>
+      <InputFied resObj={calObj} />
+      <div className="buttons">
+        <div className="btn-row">
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj(calculate(calObj, 'AC'))}
+            >
+              AC
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj(calculate({ ...calObj }, '+/-'))}
+            >
+              +/-
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj(calculate({ ...calObj }, '%'))}
+            >
+              %
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj(calculate({ ...calObj }, '÷'))}
+            >
+              ÷
+            </button>
+          </div>
+        </div>
+        <div className="btn-row">
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}7`, 10) || 7 })}
+            >
+              7
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}`, 10) || 8 })}
+            >
+              8
+            </button>
+          </div>
+          <div className="btn-col">
+            <button
+              type="button"
+              className="cal-btn"
+              onClick={() => setCalObj({ ...calObj, next: parseInt(`${calObj.next}`, 10) || 9 })}
+            >
+              9
+            </button>
+          </div>
+          <div className="btn-col">
+            <button type="button" className="cal-btn" onClick={() => setCalObj(calculate({ ...calObj }, '*'))}>
+              *
+            </button>
```

Comment:
- [ ] Great job so far. However, I noticed that when I clicked on this button, It threw the error message below. So kindly fix this :100: 
- Error message
![image](https://user-images.githubusercontent.com/102699958/228616484-b0a6e94f-8100-455b-9eb1-efbd396c6e9a.png)